charts/jenkins/VALUES_SUMMARY.md

```diff
@@ -239,12 +229,6 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `serviceAccountAgent.create`      | Configures if an agent ServiceAccount with this name should be created | `false`         |
 | `serviceAccountAgent.annotations` | Configures annotation for the agent ServiceAccount | `{}`                              |
 
-#### Deprecated
-| Parameter                         | Description                          | Default                                   |
-| --------------------------------- | ------------------------------------ | ----------------------------------------- |
-| `master.customConfigMap`          | Deprecated: Use a custom ConfigMap   | `false`                                   |
-| `master.additionalConfig`         | Deprecated: Add additional config files | `{}`                                   |
```

Comment:
any idea why `master.additionalConfig` was deprecated? it's definitely useful for getting one-off configuration onto the server which can be processed by an init script. I actually use it to work around a [Job DSL plugin issue](https://issues.jenkins-ci.org/browse/JENKINS-59971) regarding string credentials. I'm just discovering this issue has a workaround so perhaps I'll no longer need `master.additionalConfig`...but I wouldn't be surprised if others are using it for similar workarounds.

Reply:
Could you explain how you are using it?

Reply:
I like having the configuration in my values file vs managing it out of band. since it's related to folder-level credentials, I'd now rather use the Job DSL plugin workaround or `kubernetes-credentials-provider-plugin` once it adds support for folder-level credentials.

since this field has been deprecated since v0.29.0 then it's definitely due to be removed...and hopefully any other use cases for it now have better options

Reply:
Good pointer. to the PR of the credentials-provider-plugin.